packages/shared/chat/sagas.ts

```diff
@@ -119,16 +121,27 @@ function* handleSendMessage(action: SendMessage) {
     // If the message was not a command ("/cmdname"), then send message through wire
     const currentUserId = yield select(getCurrentUserId)
     if (!currentUserId) throw new Error('cannotGetCurrentUser')
```

Comment:
Who catches this throw? This would be a fatal event, shutting down the client. Prefer "logger + trackEvent + return" over uncatched throws